Readme.md

```diff
@@ -53,9 +53,8 @@ Then follow the indications on screen as they appear.
 ```powershell
 # Replace my-gameserver with how you wish to name your game
 set-item env:NODE_ENV=development
-npm install mage --create --prefix my-gameserver
-cd my-gameserver
-npm run develop
+mkdir my-gameserver && cd my-gameserver
+npx mage create
```

Comment:
Wouldn't it be nice if this could create and `cd` into the folder from this one-liner?
ie: `npx mage create my-gameserver`

Reply:
Agreed, let me see what I can do.